package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "splainer-search",
-  "version": "2.30.7",
+  "version": "2.31.7",
```

Comment:
Let's revert this back to 2.30.7..  when I cut the release, I use `np` to change the version, and it'll move to 2.31.0

Reply:
Sorry. This was inadvertent. I was testing locally with a build of Quepid. Will revert this.

Reply:
No worries!!!!    Also, look at using https://github.com/o19s/quepid/blob/main/docker-compose.override.yml.example to let you force quepid to load your locally compiled version so you don't need to bump the version, build the node package, and then update quepid.   makes your workflow much easier..  and the debugger gets the dev version of splainer-search!

Reply:
https://github.com/o19s/quepid/blob/main/README.md#debugging-splainer-and-other-npm-packages